src/Util/ObjectAclManipulator.php

```diff
@@ -26,6 +29,28 @@
  */
 class ObjectAclManipulator extends BaseObjectAclManipulator
 {
+    /**
+     * NEXT_MAJOR: Remove null.
+     *
+     * @var ManagerRegistry|null
+     */
+    private $registry;
+
+    public function __construct(?ManagerRegistry $registry = null)
```

Comment:
should we also adjust the service definition for `sonata.admin.manipulator.acl.object.orm` so it gets the manager registry as the constructor argument?

Reply:
Sure